docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -57,11 +50,11 @@ subquestion: |
 
   Field name (bold if {reserved}) | Field type | {Internal DA name} | Max length
   -----------|------------|-------------------|-----------
-  % for field in fields:
+  % for field in fields_raw:
```

Comment:
The field list screen also needs to say what the author should actually be doing. Before, the preflight tool had a specific purpose: come here to see all of the labels in your document. Now, authors come to the weaver and then are suddenly shown all their fields, without any additional information.

Reply:
Agreed. I'll leave it to #391.